api/src/restful/exceptions.py

```diff
@@ -0,0 +1,101 @@
+from fastapi import HTTPException
+from pydantic import BaseModel
+from starlette import status as request_status
+
+
+# Pydantic models can not inherit from "Exception", but we use it for openapi spec
+class ErrorResponse(BaseModel):
+    status: int = 500
+    type: str = "ApplicationException"
+    message: str = "The requested operation failed"
+    debug: str = "An unknown and unhandled exception occurred in the API"
+    data: dict = None
+
+
+class ApplicationException(Exception):
```

Comment:
can ApplicationException inherit from 2 classes?
ApplicationException(Exception, ErrorResponse)

Then the class variables status, type, message, debug, data does not have to be repeated in the ApplicationException class

Reply:
It cant, or; I could not find a way :/